services/director-v2/tests/unit/test_modules_dynamic_sidecar_docker_service_specs.py

```diff
@@ -75,22 +79,316 @@ def simcore_service_labels() -> SimcoreServiceLabels:
     )
 
 
+@pytest.fixture
+def minimal_catalog_config(project_env_devel_environment, monkeypatch):
```

Comment:
is this stil required?

Reply:
actually wanted to ask you.... I'll remove it.